packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -0,0 +1,61 @@
+import { UnavailabilityError } from 'expo-errors';
+
+import ExpoFaceDetector from './ExpoFaceDetector';
+
+export type Point = { x: number; y: number };
+
+export type FaceFeature = {
+  bounds: {
+    size: {
+      width: number;
+      height: number;
+    };
+    origin: Point;
+  };
+  smilingProbability?: number;
+  leftEarPosition?: Point;
+  rightEarPosition?: Point;
+  leftEyePosition?: Point;
+  leftEyeOpenProbability?: number;
+  rightEyePosition?: Point;
+  rightEyeOpenProbability?: number;
+  leftCheekPosition?: Point;
+  rightCheekPosition?: Point;
+  leftMouthPosition?: Point;
+  mouthPosition?: Point;
+  rightMouthPosition?: Point;
+  bottomMouthPosition?: Point;
+  noseBasePosition?: Point;
+  yawAngle?: number;
+  rollAngle?: number;
+};
+
+export type ValuesOf<T extends any[]> = T[number];
+
+export type FaceDetectorMode = ValuesOf<typeof ExpoFaceDetector.Mode>;
```

Comment:
`ExpoFaceDetector.Mode` isn't defined -- seems more useful to write `export type FaceDetectorMode = string[];` for now.